numpy/lib/polynomial.py

```diff
@@ -590,14 +606,13 @@ def polyfit(x, y, deg, rcond=None, full=False, w=None, cov=False):
     elif cov:
         Vbase = inv(dot(lhs.T, lhs))
         Vbase /= NX.outer(scale, scale)
-        # Some literature ignores the extra -2.0 factor in the denominator, but
```

Comment:
Wouldn't it be nice if this comment included a reference... cc @teoliphant who git assigns the blame to. 

In any case, should we add a note that this was different here? Just in case someone later decides that really Bayesian analysis insists there should be a `-2`... (to help ensure this doesn't happen, I also suggest a 0-D test case below - any changes from the now-corrected scaling will make that fail!)

Reply:
I added a small note about the removal of the "-2.".